examples/yarnspinner_without_bevy/Cargo.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "yarnspinner_without_bevy_examples"
+version = "0.1.0"
+edition = "2021"
+repository = "https://github.com/YarnSpinnerTool/YarnSpinner-Rust"
+homepage = "https://docs.yarnspinner.dev/"
+keywords = ["gamedev", "dialog", "yarn", "bevy"]
+categories = ["game-development", "compilers"]
+authors = ["Jan Hohenheim <jan@hohenheim.ch>"]
```

```suggestion
authors = ["Joe Clay <27cupsofcoffee@gmail.com>"]
```

Feel free to take credit! :)